extensions/3DTILES_content_gltf/README.md

```diff
@@ -36,11 +36,11 @@ This extension is required, meaning it must be placed in both the `extensionsUse
 
 This extension allows a tileset to use [glTF](https://github.com/KhronosGroup/glTF/tree/master/specification/2.0) assets directly as tile content. Both `glTF` JSON and `GLB` binary formats are supported.
 
-Using glTF as a tile format simplifies content pipelines from creation to runtime. This allows greater compatibility with existing tools (e.g. 3D modeling software, validators, optimizers) that create or process glTF assets. Runtime engines that currently support glTF can more easily support 3D Tiles.
+Using glTF as a tile format simplifies content pipelines from creation to runtime. This allows greater compatibility with existing tools (e.g. 3D modeling software, validators, optimizers) that create or process glTF assets. Runtime engines that currently support glTF can more easily support 3D Tiles. In many cases, existing tile formats can be converted into the corresponding glTF content, as described in the [Migration Guide](MIGRATION_GUIDE.md).
 
 ## Extension JSON
 
-With this extension, the tile content may be a glTF asset. Runtime engines must be able to determine compatibility before loading the content. If the glTF asset uses or requires certain glTF extensions, then these extensions must be listed on the `3DTILES_content_gltf` object. This is a property of the top-level tileset `extensions` object with the following properties:
+With this extension, the tile content may be a glTF asset. Runtime engines must be able to determine compatibility before loading the content. If the glTF asset uses or requires certain glTF extensions, then these extensions must also be listed in the `3DTILES_content_gltf` object. This is a property of the top-level tileset `extensions` object with the following properties:
 
 * `extensionsUsed`: an array of glTF extensions used by glTF content in the tileset.
 * `extensionsRequired`: an array of glTF extensions required by glTF content in the tileset.
```

Comment:
I think this wording could be more specific.

```suggestion
* `extensionsUsed`: an array of glTF extensions used by any glTF content in the tileset.
* `extensionsRequired`: an array of glTF extensions required by any glTF content in the tileset.
```

Reply:
Oh hm it seems like @donmccurdy and I have slightly different ideas of the semantics of how "content" should be described in text. (see https://github.com/CesiumGS/3d-tiles/pull/488#issuecomment-933641635.

I suppose "by glTF content" sounds natural in English if we're talking about the concept of tile contents in general, but I usually assume "content" in these specs has the specific technical meaning of whatever asset is pointed to by `tile.content.uri`. If there were more than one of them, I would say "glTF contents".

(note that the 3D Tiles 1.0 is a bit vague on the wording, sometimes "content" and "tile" are used interchangeably, though with the advent of `3DTILES_multiple_contents`, it becomes important to distinguish the concepts.)

Though overall this is a rather minor nitpick.

Reply:
Although some details about this are currently still being discussed, I think that adding "any" matches the current meaning, and thus, serves as a clarification here